workitem/event/event_repository.go

```diff
@@ -0,0 +1,119 @@
+package event
+
+import (
+	"context"
+
+	"github.com/jinzhu/gorm"
+	errs "github.com/pkg/errors"
+	uuid "github.com/satori/go.uuid"
+
+	"github.com/fabric8-services/fabric8-wit/account"
+	"github.com/fabric8-services/fabric8-wit/workitem"
+)
+
+const APIStringTypeEvents = "events"
```

Comment:
Shouldn't this be called less generic and maybe `workitemevents`?

Reply:
I don't see this is conflicting with anything.